google/generativeai/types/retriever_types.py

```diff
@@ -522,6 +524,8 @@ class Document(abc.ABC):
     name: str
     display_name: str
     custom_metadata: list[CustomMetadata]
+    create_time: str
+    update_time: str
```

Comment:
Should these be optional?

Reply:
I guess the use case would be .. if you wanted to pass one to a `create_*` method. Would that make sense?

Reply:
will keep as is